_build_cfg.yml

```diff
@@ -222,11 +222,13 @@ Topics:
     Dir: install
     Distros: openshift-origin,openshift-enterprise
     Topics:
-      - Name: Overview
-        File: index
+      - Name: Planning
+        File: planning
       - Name: Prerequisites
         File: prerequisites
-      - Name: RPM vs Containerized
+      - Name: Host Preparation
+        File: host_preparation
+      - Name: Containerized Services
```

Comment:
I know the text in the topic says "install OCP as a set of services within containers", but I think "Services" is too broad a word here (plus "service" is an overloaded word in OpenShift). How about:

Containerized Components
Containerized Infrastructure

....mmm, that's all I got right now.